src/views/Stack/Card.tsx

```diff
@@ -327,22 +338,9 @@ export default class Card extends React.Component<Props> {
         set(this.isSwiping, FALSE),
         this.runTransition(
           cond(
-            or(
-              and(
-                greaterThan(abs(this.gesture), SWIPE_DISTANCE_MINIMUM),
-                greaterThan(
-                  abs(this.velocity),
-                  SWIPE_VELOCITY_THRESHOLD_DEFAULT
-                )
-              ),
-              cond(
-                greaterThan(
-                  abs(this.gesture),
-                  SWIPE_DISTANCE_THRESHOLD_DEFAULT
-                ),
-                TRUE,
-                FALSE
-              )
+            greaterThan(
+              abs(this.extrapolatedPosition),
+              divide(this.layout.width, 2)
```

Comment:
I think we need to use `this.distance` here to support both vertical and horizontal gestur

```suggestion
              divide(this.distance, 2)
```